klarna-on-demand/src/main/java/com/klarna/ondemand/Crypto.java

```diff
@@ -0,0 +1,12 @@
+package com.klarna.ondemand;
+
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SignatureException;
+
+public interface Crypto {
```

Comment:
Does Java not have the same convention as other languages where interfaces have a very indicative name? Something like C#'s notion of `ICrypto`?